.github/workflows/pre-submit.units.yml

```diff
@@ -159,6 +159,24 @@ jobs:
           # Run yamllint
           make yamllint
 
+  generate:
```

Comment:
Is the generation step really necessary? Maybe I can just maintain the Go code in `langconfig.go` rather than have `langconfig.yml` be the source of truth. That would eliminate the duplication and the need for code generation.